scripts/generate-pem.js

```diff
@@ -0,0 +1,3 @@
+/* istanbul ignore file */
+
+require('https-pem/install')
```

Comment:
Is it possible to run this only on environments that set ignore scripts command?

Reply:
How could we detect such environment? And is it worth investing into? It's not like this adds seconds of additional time to the test suite.

Reply:
You're right.